bedrock/firefox/views.py

```diff
@@ -737,11 +738,14 @@ class FirefoxFocusView(BlogPostsView):
 
 
 class FirefoxHubView(BlogPostsView):
-    blog_posts_limit = 1
-    blog_posts_template_variable = 'articles'
-    blog_slugs = 'firefox'
-    blog_tags = ['home']
-    template_name = 'firefox/hub/home.html'
+    if switch('firefox-57-release'):
```

Comment:
It would be nice to have some unit tests that ensure the correct template is loaded for both states of the switch